contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +114,67 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+== Anchoring titles and section headings
+
+All titles and section headings must have an anchor. The anchor ID must be similar to the title or section heading.
+
+=== Anchoring in assembly files
+
+The following is an example anchor in an assembly file:
```

Comment:
Should this say that this is an example of an anchor id (rather than an anchor).

Reply:
Thanks for highlighting this. I have now updated all instances of "anchor" in the PR to state "anchor ID", for consistency.